tests/unit/test_slide.py

```diff
@@ -277,6 +278,80 @@ def but_it_raises_expection_when_level_does_not_exist(self, level, tmpdir):
             == f"Level {level} not available. Number of available levels: 1"
         )
 
+    @pytest.mark.parametrize(
+        "fixture_slide, level, expected_value",
+        (
+            pytest.param(
+                EXTERNAL_SVS.LIVER_1,
+                0,
+                "20X",
+                marks=pytest.mark.skipif(not on_ci(), reason="To run only on CI"),
+            ),
+            pytest.param(
+                EXTERNAL_SVS.LIVER_1,
+                1,
+                "5.0X",
+                marks=pytest.mark.skipif(not on_ci(), reason="To run only on CI"),
+            ),
+            pytest.param(
+                EXTERNAL_SVS.LIVER_1,
+                2,
+                "1.25X",
+                marks=pytest.mark.skipif(not on_ci(), reason="To run only on CI"),
+            ),
+        ),
+    )
+    def it_knows_its_magnification_factors(self, fixture_slide, level, expected_value):
+        slide = Slide(fixture_slide, "")
+
+        magnification_factor = slide.level_magnification_factor(level=level)
+
+        assert magnification_factor == expected_value
```

Comment:
these tests should go with the integration tests as we are using real slides, and unit tests for this method are missing :)